modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -7,9 +7,10 @@
         :size="icon ? 16 : 12"
       />
     </span>
-    <span class="apos-notification__label">
-      {{ label }}
-    </span>
+    <span
+      class="apos-notification__label" v-html="label"
```

Comment:
See above, recommending a new and separate beta ticket for reconsideration on this since Stu is on record saying he prefers more flexibility (I originally suggested what you're suggesting).